metadata.rb

```diff
@@ -9,6 +9,7 @@
 depends          'sudo'
 depends          'java'
 depends          'apache2'
+depends          'yum', '~> 3.10.0'
```

Comment:
yum_repository is built into chef now if you assume the user is on a later version of chef 12